thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -83,6 +85,8 @@ export default Controller.extend({
 
   sessionService: service('rootcause-session-datasource'),
 
+  anomalyFunctionService: service('rootcause-anomalyfunction-cache'),
+
   //
```

Comment:
Something missing here?

Reply:
interesting. what creates the impression that something is missing here? I mean, what would you expect to be there?